ee/tables/execparsers/json/json.go

```diff
@@ -0,0 +1,21 @@
+//go:build darwin || linux
+// +build darwin linux
```

Comment:
Why is this darwin/linux only? (Usually, unless we aren't able to, we keep files cross-platform)